helpers/setupjsongenerator.py

```diff
@@ -0,0 +1,675 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import os
+import json
+from ci.autotests import AutoTests
+from ovs.dal.hybrids.albabackend import AlbaBackend
+from ovs.dal.hybrids.diskpartition import DiskPartition
+from ovs.lib.helpers.toolbox import Toolbox
+
+
+
+class SetupJsonGenerator(object):
+    """
+    Class to automate construction of a setup.json file.
+    Attributes:
```

Comment:
Instead of listing the methods, I'd describe what the use case of this class is. (why is it useful for example)